x-pack/solutions/security/plugins/security_solution/public/plugin.tsx

```diff
@@ -443,7 +443,10 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
     // When the user does not have any of the capabilities required to access security solution, the plugin should be inaccessible
     // This is necessary to hide security solution from the selectable solutions in the spaces UI
-    if (!hasAccessToSecuritySolution(capabilities)) {
+    if (
+      !hasAccessToSecuritySolution(capabilities) &&
+      !capabilities.securitySolutionCasesV3?.read_cases
```

Comment:
Had to revert this change, because of an edge case where redirect to cases takes place: 
https://github.com/elastic/kibana/blob/1b167d9dc23a9e0e8e47992a37563ca89ccf3c7d/x-pack/solutions/security/plugins/security_solution/public/app/app_routes.tsx#L36